app/celery/twilio_tasks.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime, timedelta, timezone
+
+from flask import current_app
+from sqlalchemy import select
+
+from app import db, notify_celery, twilio_sms_client
+from app.celery.exceptions import NonRetryableException
+from app.constants import NOTIFICATION_STATUS_TYPES_COMPLETED
+from app.models import Notification
+
+
+def _get_notifications() -> list:
+    """Returns a list of notifications not in final state."""
+
+    current_app.logger.info('Getting notifications to update status')
+    one_hour_ago = datetime.now(timezone.utc) - timedelta(hours=1)
+    query = (
+        select([Notification])
+        .where(Notification.notification_type == 'sms')
+        .where(Notification.sent_by == 'twilio')
+        .where(~Notification.status.in_(NOTIFICATION_STATUS_TYPES_COMPLETED))
+        .where(Notification.created_at < one_hour_ago)
```

Comment:
should this be greater than one_hour_ago?

Reply:
No, the variable one_hour_ago is one hour in the past, so we're looking for things less than that. Greater than would be newer.